microreactors/gcmr/core/MESH/Griffin_mesh.i

```diff
@@ -0,0 +1,650 @@
+    background_block_ids = '10'
+    polygon_size = 1
+    polygon_size_style ='apothem'
+    ring_radii = '0.85'
+    ring_intervals = '2'
+    ring_block_ids = '401 400'
+    preserve_volumes = on
+    #quad_center_elements = true
+    #external_boundary_id = 100
+    #interface_boundary_id_shift = 100
+  []
+
+  [TRISO_fuel_mid]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '2 2 2 2 2 2'
+    background_intervals = 1
+    background_block_ids = '10'
+    polygon_size = 1
+    polygon_size_style ='apothem'
+    ring_radii = '0.85'
+    ring_intervals = '2'
+    ring_block_ids = '4001 4000'
+    preserve_volumes = on
+    #quad_center_elements = true
+    #external_boundary_id = 100
+    #interface_boundary_id_shift = 100
+  []
+
+    [TRISO_fuel_out]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '2 2 2 2 2 2'
+    background_intervals = 1
+    background_block_ids = '10'
+    polygon_size = 1
+    polygon_size_style ='apothem'
+    ring_radii = '0.85'
+    ring_intervals = '2'
+    ring_block_ids = '40001 40000'
+    preserve_volumes = on
+    #quad_center_elements = true
+    #external_boundary_id = 100
+    #interface_boundary_id_shift = 100
+  []
+
+  [Poison_LBP0]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '2 2 2 2 2 2'
+    background_intervals = 1
+    background_block_ids = '10'
+    polygon_size = 1
+    polygon_size_style ='apothem'
+    ring_radii = '0.25'
+    ring_intervals = '2'
+    ring_block_ids = '501 500'
+    preserve_volumes = on
+    #quad_center_elements = true
+    #external_boundary_id = 100
+    #interface_boundary_id_shift = 100
+  []
+
+  [Poison_LBP1]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '2 2 2 2 2 2'
+    background_intervals = 1
+    background_block_ids = '10'
+    polygon_size = 1
+    polygon_size_style ='apothem'
+    ring_radii = '0.25'
+    ring_intervals = '2'
+    ring_block_ids = '5001 5000'
+    preserve_volumes = on
+    #quad_center_elements = true
+    #external_boundary_id = 100
+    #interface_boundary_id_shift = 100
+  []
+
+    [centralFA]
+    type = PatternedHexMeshGenerator
+    inputs = 'YH_pin Coolant_hole  TRISO_fuel_in Poison_LBP0  Control_hole  Poison_LBP1   TRISO_fuel_out  TRISO_fuel_mid'
+    #           0          1            2           3          4            5             6               7
+    hexagon_size = 10.4
+    pattern =
+                      '3  1  2  2  1  3;
+                    1  0  2  1  2  0  1;
+                  2  2  1  2  2  1  2  2;
+                2  1  2  3  1  3  2  1  2;
+              1  2  2  1  2  2  1  2  2  1;
+            3  0  1  3  2  1  2  3  1  0  3;
+            1  2  2  1  2  2  1  2  2  1;
+            2  1  2  3  1  3  2  1  2;
+            2  2  1  2  2  1  2  2;
+            1  0  2  1  2  0  1;
+            3  1  2  2  1  3'
```

Comment:
Make this into a hexagonal pattern similar to the top half of the pattern?

Reply:
Done.